src/index.js

```diff
@@ -1,10 +1,33 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
+import { CookiesProvider } from 'react-cookie';
+import { Route, Switch, BrowserRouter as Router } from 'react-router-dom';
+import SignupView from './SignupView';
+import SigninView from './SigninView';
+import { createGlobalStyle } from 'styled-components';
+
+const GlobalStyle = createGlobalStyle`
+  display: flex;
+
+  #root {
+    margin:0;
+    display:flex;
+  }
+`;
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <CookiesProvider>
+      <GlobalStyle />
+      <Router>
+        <Switch>
+          <Route exact path="/" render={() => <SigninView />} />
```

Comment:
```suggestion
          <Route exact path="/"><SigninView /></Route>
```
render prop 사용하는것보단 child component로 넣는 것이 나중에 코드가 길어지면 보기 편했던 경험이 있어서 추천드려요